apps/ledger-live-desktop/src/renderer/reducers/settings.js

```diff
@@ -381,7 +381,7 @@ export const deepLinkUrlSelector = (state: State) => state.settings.deepLinkUrl;
 export const counterValueCurrencyLocalSelector = (state: SettingsState): Currency =>
   findCurrencyByTicker(state.counterValue) || getFiatCurrencyByTicker("USD");
 
-export const counterValueCurrencySelector: OutputSelector<State, void, Currency> = createSelector(
+export const counterValueCurrencySelector = createSelector(
```

Comment:
What's the reason for this change? TS complaining about it?

Reply:
Yes TS complaning. I did this changed based on this PR comment: https://github.com/LedgerHQ/ledger-live/pull/1513#discussion_r991531248